service_catalog/api/views/approval_workflow_api_views.py

```diff
@@ -49,7 +50,7 @@ def get_survey(self, request, pk=None):
         target_request = get_object_or_404(Request, id=pk)
         if target_request.approval_workflow_state.current_step is None:
             return Response({"error": "No pending step to approve"}, status=HTTP_404_NOT_FOUND)
-        if not request.user.has_perm(target_request.approval_workflow_state.current_step.approval_step.permission,
+        if not request.user.has_perm(target_request.approval_workflow_state.current_step.approval_step.permission.get_permission_str,
```

Comment:
get_permission_str is a method but it should be a property IMO (permission_str without get)

Reply:
Ok for that

Reply:
Parenthesis are missing here